crates/viewer/re_renderer/shader/lines.wgsl

```diff
@@ -370,10 +370,12 @@ fn fs_main_picking_layer(in: VertexOut) -> @location(0) vec4u {
 @fragment
 fn fs_main_outline_mask(in: VertexOut) -> @location(0) vec2u {
     // Output is an integer target, can't use coverage therefore.
-    // But we still want to discard fragments where coverage is low.
-    // Since the outline extends a bit, a very low cut off tends to look better.
+    // But we still want to discard fragments where coverage is too low, otherwise
+    // we'd not handle rounded corners etc. correctly.
+    // Note that `compute_coverage` may already give coverage values below 1.0 along the
+    // "main body of the line", not just the caps.
     var coverage = compute_coverage(in);
-    if coverage < 1.0 {
+    if coverage <= 0.0 {
```

Comment:
Should the same fix be applied anywhereelse?

<img width="419" alt="Screenshot 2024-12-04 at 22 25 00" src="https://github.com/user-attachments/assets/bb1f1342-c799-41c5-9795-a91c85f4ada1">

Reply:
Hmmmmmmmm likely! I'll do a bit of A/B testing on points as well (note to self: proving this on regular points should extend just fine to depth)

More note to self: This blew up because we go down in coverage along the line now when we actually have to rely on "normal" MSAA to do what little anti aliasing we can do for outlines since alpha-to-coverage doesn't work with integer targets (gosh alpha-to-coverage is such a bizarre hack!). So I'm actually not 100% sure why I previously came to the seemingly opposite conclusion on what a good coverage cutoff is. The new one makes much more sense to me given this situation.
Should enhance that comment a little bit more with this knowledge